test/config-next/ra.json

```diff
@@ -0,0 +1,73 @@
+{
+  "ra": {
+    "rateLimitPoliciesFilename": "test/rate-limit-policies.yml",
+    "maxConcurrentRPCServerRequests": 16,
+    "maxContactsPerRegistration": 100,
+    "dnsTries": 3,
+    "debugAddr": "localhost:8002",
+    "hostnamePolicyFile": "test/hostname-policy.json",
+    "maxNames": 1000,
+    "doNotForceCN": true,
+    "reuseValidAuthz": true,
+    "vaService": {
+      "serverAddresses": ["boulder:9092"],
+      "serverIssuerPath": "test/grpc-creds/ca.pem",
+      "clientCertificatePath": "test/grpc-creds/client.pem",
+      "clientKeyPath": "test/grpc-creds/key.pem",
+      "timeout": "90s"
+    },
+    "amqp": {
+      "serverURLFile": "test/secrets/amqp_url",
+      "insecure": true,
+      "serviceQueue": "RA.server",
+      "VA": {
+        "server": "VA.server",
+        "rpcTimeout": "60s"
+      },
+      "SA": {
+        "server": "SA.server",
+        "rpcTimeout": "15s"
+      },
+      "CA": {
+        "server": "CA.server",
+        "rpcTimeout": "15s"
+
```

Comment:
nit: This whitespace was in the original as well but seems out of place - can you nix it?